Checkout/Source/Tokenisation/CheckoutAPIService.swift

```diff
@@ -41,7 +50,18 @@ final public class CheckoutAPIService: CheckoutAPIProtocol {
     let tokenRequestFactory = TokenRequestFactory(cardValidator: cardValidator, decoder: snakeCaseJSONDecoder)
     let tokenDetailsFactory = TokenDetailsFactory()
     let logManager = LogManager.self
-
+      
+    var riskEnvironment: RiskEnvironment
+    switch environment {
+    case .production:
+      riskEnvironment = .production
+    case .sandbox:
+      riskEnvironment = .sandbox
+    }
```

Comment:
```suggestion 
  let environmentMapping: [BaseURLProviding: RiskEnvironment] = [.production: .production, .sandbox: .sandbox]
```

then below: 
`let riskConfig = RiskConfig(publicKey: publicKey, environment: environmentMapping[environment], framesMode: true)